p2p/speedmeter_test.go

```diff
@@ -10,10 +10,48 @@ import (
     "time"
 )
 
-func Test_speedMeter(t *testing.T) {
-    sp := NewSpeedMeter(100, 10)
-    sp.Feed(128 * 1024)
-    time.Sleep(300)
-    sp.Feed(128 * 1024)
-    fmt.Println("speedMeter:", sp.GetRate())
+func Test_NewSpeedMeter(t *testing.T) {
+    var step uint64 = 5
+    var itemsNum uint = 10
+    v1 := NewSpeedMeter(step, itemsNum)
+
+    if v1.step != step || v1.itemsNum != itemsNum {
```

Comment:
use check.Assert?

Reply:
Currently I use testing.T.  we can refine it to check later.